.coveragerc

```diff
@@ -1,6 +1,6 @@
 
 [run]
-branch = False
+branch = True
```

Comment:
Are we still enabling branch coverage?

Reply:
that comment is against outdated code, try refreshing.
I re-disabled branch coverage as agreed upon, it is deferred til SPARK-87